scripts/bfasst/compare_waveforms/Templates/get_paths.py

```diff
@@ -0,0 +1,42 @@
+import bfasst
+from bfasst.compare_waveforms.Templates import structs
+
+paths = structs.paths
+
+def get_paths(self, design):
```

Comment:
get_paths isn't part of a class, so it shouldn't require self.  If it does, it should be part of a class.

Reply:
I think I mentioned this in another comment, but if passing in self.work_dir would work, that would probably be the easiest solutions since it's the only actual usage of self in this file.